doc/adoc/user_guide.adoc

```diff
@@ -0,0 +1,117 @@
+Code available: {Repo}
+
+== Concept
+The Distribution Migration System provides an upgrade path for an installed SUSE Linux Enterprise system from on major distribution to another, for example, from SUSE Linux Enterprise Server 12 SP4 to SUSE Linux Enterprise Server 15. For system updates from one Service Pack (SP) to another within a given major distribution release series the existing
+
+[listing]
+zypper migration
+
+functionality is the supported upgrade path. The distribution migration system provides the ability to migrate across major distributions without the need to use the next major version installation media to perform the system migration.
+
+The migration between two major distribution versions requires the system to be migrated to be off line to avoid system inconsistencies that may leave the system in a state that does not allow recovery. This behavior is implemented using a Live Migration Image.
+
+The distribution migration system provides the live image and a component that modifies the bootloader configuration such that on the next boot the system will boot into the migration image. Once booted into the migration live image the following chain of services will be executed:
+
+- detect the system to be migrated
+- mount the necessary file systems
+- setup the network to match the network configuration of the system to be migrated
+- prepare the package manager for the migration task
+- migrate the system
+- update the boot loader configuration
+- unmount all mounted file systems
+- reboot
+
+Should an error occur prior to the start of the actual migration the system will be returned to it's original state.
+
+== Requirements
+The system to be migrated must be registered. Pay as you go instances in the Public Cloud are automatically registered to the SUSE operated update infrastructure. All other systems must be connected to the SUSE Customer Center (SCC), a Subscription Management Tool (SMT), or a Repository Management tool (RMT) server. For system managed via SUSE Manager use the migration path provided by SUSE Manager. 
+
+== Recommendation
+During the migration it is not possible to log into the system in any other way then ssh key based login. If your system does not have such a configuration it is recommended that at least one of the users on the system has a '~/.ssh/authorized_keys' file with a private key accessible by the person executing the system migration.
+
+== Installation
+
+The distribution migration system is made available in the Public Cloud module. Therefore this module has to be enable on the system to be migrated. For running on-demand instances this module is already enabled in the instance.
+
+For data center customers it is recommended to continue to use the documented offline distribution migration using the next distribution version installation media.
+
+To install the distribution migration system execute as root
+
+[listing]
+$ zypper in SLE15-Migration suse_migration_activation
```

Comment:
The activation package name is: suse-migration-sle15-activation

Reply:
Fixed.